examples/gno.land/p/samourai/daocond/daocond_test.gno

```diff
@@ -0,0 +1,119 @@
+package daocond_test
+
+import (
+	"errors"
+	"testing"
+
+	"gno.land/p/demo/urequire"
+	"gno.land/p/samourai/daocond"
+)
+
+func TestCondition(t *testing.T) {
+	dao := newMockDAO()
+
+	// leaf conditions
+	membersMajority := daocond.MembersThreshold(0.6, dao.isMember, dao.membersCount)
+	publicRelationships := daocond.RoleCount(1, "public-relationships", dao.hasRole)
+	financeOfficer := daocond.RoleCount(1, "finance-officer", dao.hasRole)
+
+	urequire.Equal(t, "60% of members", membersMajority.Render())
+	urequire.Equal(t, "1 public-relationships", publicRelationships.Render())
+	urequire.Equal(t, "1 finance-officer", financeOfficer.Render())
+
+	// ressource expressions
+	ressources := map[string]daocond.Condition{
+		"social.post":    daocond.And(publicRelationships, membersMajority),
+		"finance.invest": daocond.Or(financeOfficer, membersMajority),
+	}
+
+	urequire.Equal(t, "[1 public-relationships AND 60% of members]", ressources["social.post"].Render())
+	urequire.Equal(t, "[1 finance-officer OR 60% of members]", ressources["finance.invest"].Render())
+
+}
+
```

Comment:
I think we should add back the `TestState` test and rename `TestEval`
Or at least remove the utils bellow that are unused now

Reply:
[1b6e526](https://github.com/samouraiworld/gno/pull/3/commits/1b6e52697be45937cec0611b2302fe1f13aacf71)